include/openthread/platform/radio.h

```diff
@@ -856,6 +856,17 @@ bool otPlatRadioIsCoexEnabled(otInstance *aInstance);
  */
 otError otPlatRadioGetCoexMetrics(otInstance *aInstance, otRadioCoexMetrics *aCoexMetrics);
 
+/**
+ * The radio driver calls this method to notify OpenThread that an ACK is
```

Comment:
For header IE, it could be added to Enh-ACK or data frame. For example, a SSED send out a data frame to its parent could include CSL IE in it. This data frame also needs this callback to update CSL IE and do the CCM*. So I think it might be better to rename this function and related callback to a more general name, like `otPlatRadioTxStarted`. Thoughts?

Reply:
I agree to use a general function to do the TxStart update for both data and ack. However, there's already a function with that name: https://github.com/openthread/openthread/blob/master/src/core/radio/radio_platform.cpp#L53 .
And that function is used to start a timer to monitor ACK timeout. What about we call this method `otPlatRadioTxInterruptHandle`?

Reply:
Sounds good. `otPlatRadioTxStarted` was used in none-interrupt context. Maybe you can just name this API to specify it is to handle Tx Started interrupt in ISR context.